src/main/java/frc/robot/common/ControlPanelColorSensor.java

```diff
@@ -12,7 +12,8 @@
         RED(new Color(0.325, 0.453, 0.221)),
         GREEN(new Color(0.269, 0.550, 0.181)),
         BLUE(new Color(0.175, 0.469, 0.356)),
-        YELLOW(new Color(0.188, 0.503, 0.310));
+        YELLOW(new Color(0.188, 0.503, 0.310)),
+        UNSET(null);
```

Comment:
I would actually make this `UNKNOWN` rather than `UNSET` because without context, if a color isn't one of those it's uknown. Only in the context of the driverstation sending a color is it unset. It'll also make sense if you return `UNKNOWN` when the confidence for the color matcher is low, whereas `UNSET` wouldn't make as much sense in that context

Reply:
And I think you should give it a default value of either `0,0,0` or `1,1,1`. Setting null *will* cause an NPE when the color matcher tries to match to it.